src/main/java/net/spy/memcached/ArcusClient.java

```diff
@@ -336,6 +349,53 @@ public ArcusClient(ConnectionFactory cf, List<InetSocketAddress> addrs)
     this(cf, DEFAULT_ARCUS_CLIENT_NAME + "-" + CLIENT_ID.getAndIncrement(), addrs);
   }
 
+  @SuppressWarnings("removal")
+  private static boolean validateDnsCacheTtl() {
+    Integer securityDnsTtl = parseDnsTtl(() -> Security.getProperty(DNS_CACHE_TTL_SECURITY_PROP),
+            "Can not access to dns cache ttl property in security manager.");
+    if (securityDnsTtl != null) {
+      return isValidTtl(securityDnsTtl);
+    }
+
+    Integer systemDnsTtl = parseDnsTtl(() -> System.getProperty(DNS_CACHE_TTL_SYSTEM_PROP),
+            "Can not access to dns cache ttl property in system manager.");

```

Comment:
`Can not access to dns cache ttl property in system manager.` => `Can not access to dns cache ttl in system properties.`